main/acle.md

```diff
@@ -7499,6 +7653,1415 @@ This is because the ACLE intrinsic calls do not imply a particular
+
+    s_callback = n_callee;    // Ill-formed
+    s_callback = s_callee;    // OK
+    s_callback = sc_callee;   // Ill-formed
+
+    sc_callback = n_callee;   // Ill-formed
+    sc_callback = s_callee;   // Ill-formed
+    sc_callback = sc_callee;  // OK
+  }
+```
+
+A type that has both an [`arm_streaming`](#arm_streaming) attribute and
+an [`arm_streaming_compatible`](#arm_streaming_compatible) attribute is
+[ill-formed](#ill-formed).
+
+### Effect of streaming mode on VL
+
+The value returned by an SVE intrinsic like `svcntb` depends on
+whether or not the [abstract machine](#abstract-machine) is in streaming mode:
+if the abstract machine is in streaming mode then the returned value
+depends on the “streaming vector length” (SVL), otherwise the returned
+value depends on the non-streaming vector length.
+
+The runtime size of SVE vector and predicate types varies in the same way.
+For example, an `svint8_t` object created while the abstract machine
+is in streaming mode will have SVL bits, whereas an `svint8_t` object
+created while the abstract machine is in non-streaming mode will have
+the size specified by the non-streaming vector length.
+
+The following definitions are useful when describing the consequences
+of this behavior on the vector length:
+
+*   <!-- Do not remove the following `span`, it is needed to create an
+anchor that can be referred via an internal hyperlink to the paragraph
+following it. --><span id="vl-dependent"></span>
+    A type T is said to be “VL-dependent” if an object of type T created
+    while the [abstract machine](#abstract-machine) is in streaming mode
+    has a different size from an object of type T created while the abstract
+    machine is in non-streaming mode.
+
+    Pointer and reference types are never VL-dependent. No types are
+    VL-dependent if the streaming and non-streaming vector lengths are equal.
+
+*   As a shorthand, an object or value is said to be VL-dependent if its
+    type is VL-dependent.
+
+    Like normal objects, a VL-dependent object never changes size after
+    it has been created. The “VL-dependent” classification instead
+    decides when it is valid to access the object, as described below.
+
+*   <!-- Do not remove the following `span`, it is needed to create an
+anchor that can be referred via an internal hyperlink to the paragraph
+following it. --><span id="streaming-object"></span>
+    A VL-dependent object created while the abstract machine is in
+    streaming mode is called a “streaming object”.
+
+*   <!-- Do not remove the following `span`, it is needed to create an
+anchor that can be referred via an internal hyperlink to the paragraph
+following it. --><span id="non-streaming-object"></span>
+    A VL-dependent object created while the abstract machine is in
+    non-streaming mode is called a “non-streaming object”.
+
+If any of the following events occurs during the execution of a program
+then the behavior is undefined:
```

Comment:
I've added the comma before “then”, but the intention was to refer to a singular event rather than a collection of events.  The wording is trying to match the style of the C/C++ standard.